server/pages/index.js

```diff
@@ -104,15 +116,8 @@ function generateStaticUrls( request ) {
 		urls[ file.path ] = getUrl( file.path, file.hash );
 	} );
 
-	const assets = request.app.get( 'assets' );
-
-	assets.forEach( function( asset ) {
-		const name = asset.name;
-		urls[ name ] = asset.url;
-		if ( config( 'env' ) !== 'development' ) {
```

Comment:
need this bit too to generate the `-min` variants to hand off to the jade template

Reply:
eh. it feels dumb to need that logic in two places.  I'll just place it here and not have the jade template care.